src/recipes/dom-pedro.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Dom Pedro",
+  "description": "Dom Pedro is a deliciously creamy drink, this particular recipe is a bit spiced up, so it packs a good punch!",
+  "github": "jessicavrooyen",
+  "ingredients": [
+      "2 Tots Amarula Cream",
+      "1 Tot Frangelico",
+      "150 ml Vanilla Ice-cream",
+      "25 ml Coconut cream",
+      "Pinch of cinnamon",
+      "1-2 Blocks of grated chocolate"
+  ],
+  "directions": [
+      "1. Add your ice-cream, coconut cream, and amarula cream into a blender.",
+      "2. Add a few cubes of ice (optional)",
+      "3. Blend it all up & pour into glass",
+      "4. Drizzle Frangelico on top",
+      "5. Garnish with cinnamon and chocolate"
```

Comment:
Numbers not necessary as this list is rendered as an ordered list, which will prepend the numeration by default. Can you remove these, please?